thirdeye/thirdeye-frontend/app/pods/components/range-pill-selectors/template.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="te-pill-selectors__wrapper">
+  {{!-- Date range selector --}}
+  <div class="te-pill-selectors__title">{{title}}</div>
+  <ul class="te-pill-selectors">
+    {{#each timeRangeOptions as |range|}}
+      <li class="te-pill-selectors__item {{if range.isActive "te-pill-selectors__item--active"}}" {{action "onRangeOptionClick" range}} data-value={{range.value}}>
```

Comment:
This will add a whitespace if `range.isActive` is false. We can do
```
<li class="te-pill-selectors__item" class={{unless range.isActive "te-pill-selectors__item--active"}} {{action "onRangeOptionClick" range}} data-value={{range.value}}>
```

Ember will unified the class attributes.